### PR TITLE
Change class of FCM category slug to remove extra spacing

### PR DESCRIPTION
**Pull request by Scotchester**
There's too much spacing between the FCM category slug and the FCM heading. In accordance with the updated markup recommendations in cfpb/capital-framework#320, this PR fixes that.

## Changes

- FCM category markup from `p.h4` to `div.category-slug`
- Some minor CHANGELOG cleanup

## Testing

1. Pull branch
1. Reload a local page with an FCM

## Review

- @jimmynotjim 
- @anselmbradford 
- @KimberlyMunoz 
- @sebworks 

## Screenshots

See cfpb/capital-framework#320

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -16,23 +16,25 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
 ## Unreleased
 
 ## Added
-
+- Ability to use Social Media molecule as a Wagtail module in the Sidefoot
```
Comment (jimmynotjim) [inline CHANGELOG.md]: Lots of unrelated changes to the log, are they all making up for existing changes that weren't documented?

**Comment by jimmynotjim** [comment]
You'll want to update the CF dependency and re-shrinkwrap to get the margin changes you made in CF#320

File: CHANGELOG.md (hunk 1 above)
Comment (Scotchester) [inline CHANGELOG.md]: Moved this to the correct position. The merge logic didn't keep it in the Unreleased section after several releases happened while the PR containing this change waited to be merged.

File: CHANGELOG.md [hunk 2]
```diff
@@ -16,23 +16,25 @@ Given the `MAJOR.MINOR.PATCH` pattern, here is how we decide to increment:
...
+- Made Text Introduction's `has_rule` option have an effect
+- Tidied up some of the template logic around using `render_block`
+- Changed class of FCM category slug to remove extra spacing
 
 ### Removed
 - Event RSVP email link button
 - `atomicName` parameter from `checkDom` atomic helper.
 
 ### Fixed
-- get_browsefilterable_posts() call to get_page_set
```
Comment (Scotchester) [inline CHANGELOG.md]: Other than this line, which I didn't do intentionally and can't explain, the rest of the changes in this file are whitespace.

**Comment by Scotchester** [comment]
This update doesn't actually require the update to CF, but if that gets merged and released before this, I can do that.

**Comment by jimmynotjim** [comment]
Ah ok, in that case 👍 from me

File: CHANGELOG.md (hunk 2 above)
Comment (Scotchester) [inline CHANGELOG.md]: Confirmed that this line is below in the 3.0.0-3.3.20 release.

**Comment by Scotchester** [comment]
:shipit:
